SIL.WritingSystems/Resources/alltags.txt

```diff
@@ -1,2191 +0,0 @@
-*aa = aa-Latn | *aa-ET = aa-Latn-ET
```

Comment:
Daniel, I'm vaguely assuming you're on top of this since you're part of the Keyman team, but just in case, note that the following Keyman files refer to and/or access this file:
\resources\standards-data\alltags\readme.md
windows\src\buildtools\build_standards_data\Keyman.System.BuildLibPalasoAllTags.pas
windows\src\buildtools\build_standards_data\Makefile

Reply:
I wasn't aware of that but Keyman gets it from the NRSI github rather than libpalaso so there isn't a problem. I'm making a minor PR to rename the class to make that explicit.